### PR TITLE
[ADD] l10n_co_pos: DIAN Sequence for pos receipt

### PR DESCRIPTION
**Pull request by mgh-odoo**
Legal requirement for Colombia to display DIAN sequence on the receipt
Task-ID: 2169664

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/l10n_co_pos/views/views.xml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <record id="pos_config_view_form" model="ir.ui.view">
+        <field name="name">pos.config.form.view.inherit.l10n_co_pos</field>
+        <field name="model">pos.config</field>
+        <field name="inherit_id" ref="point_of_sale.pos_config_view_form"/>
+        <field name="arch" type="xml">
+            <xpath expr="//div[@id='order_reference']" position="attributes">
```
Comment (pimodoo) [inline addons/l10n_co_pos/views/views.xml]: the field order_reference does not exist remove this xpath

File: addons/l10n_co_pos/static/src/js/pos.js [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+"use strict";
+
+var models = require('point_of_sale.models');
+
+var _super_posmodel = models.PosModel.prototype;
+models.PosModel = models.PosModel.extend({
+    is_colombian_country: function () {
+        return this.company.country.code === 'CO';
+    },
+    _save_to_server: function () {
```
Comment (pimodoo) [inline addons/l10n_co_pos/static/src/js/pos.js]: you should not override,  _save_to_server function, but the 'post_push_order_resolve' in paymentScreen (like for french localisation) and do the rpc that will get the "name" and set it as l10n_co_dian.

File: addons/l10n_co_pos/models/pos_order.py [hunk 3]
```diff
@@ -0,0 +1,14 @@
+# -*- coding: utf-8 -*-
+
+from odoo import models, fields, api
+
+
+class PosOrder(models.Model):
+    _inherit = 'pos.order'
+
+    @api.model
+    def create_from_ui(self, orders, draft=False):
```
Comment (pimodoo) [inline addons/l10n_co_pos/models/pos_order.py]: no need of modifying the return of create from UI, as an rpc will be done in the override of "post_push_order_resolve"

File: addons/l10n_co_pos/views/views.xml (hunk 1 above)
Comment (mgh-odoo) [reply addons/l10n_co_pos/views/views.xml]: I've used ID 'order_reference' to remove group_no_one

File: addons/l10n_co_pos/static/src/js/pos.js (hunk 2 above)
Comment (mgh-odoo) [reply addons/l10n_co_pos/static/src/js/pos.js]: to avoid extra RPC call I did override _save_to_server, by the way, changes have been done. :+1:

**Comment by pimodoo** [comment]
robodoo r+ rebase-ff

**Pull request closed without merging**